From 1a9920fdbd1d248b211900dee24133fdd3dadd88 Mon Sep 17 00:00:00 2001
From: Karthik Kambatla <karthik@kambatla.com>
Date: Fri, 14 Feb 2014 15:46:33 -0800
Subject: [PATCH 129/596] MR1: CLOUDERA-BUILD. Update version to
 2.3.0-mr1-cdh5.0.0-SNAPSHOT

(cherry picked from commit 526a8f43dedb63692745aedd44e9326f2adc7c94)
(cherry picked from commit 533a29b47dfdefec02e06ffa1dd2877c67e301f5)
---
 hadoop-mapreduce1-project/build.xml                |    2 +-
 hadoop-mapreduce1-project/cloudera-pom.xml         |    4 ++--
 .../cloudera/maven-packaging/hadoop-client/pom.xml |    4 ++--
 .../cloudera/maven-packaging/hadoop-core/pom.xml   |    4 ++--
 .../maven-packaging/hadoop-examples/pom.xml        |    4 ++--
 .../maven-packaging/hadoop-minicluster/pom.xml     |    4 ++--
 .../cloudera/maven-packaging/hadoop-mrunit/pom.xml |    4 ++--
 .../maven-packaging/hadoop-streaming/pom.xml       |    4 ++--
 .../maven-packaging/hadoop-tarball/pom.xml         |    4 ++--
 .../cloudera/maven-packaging/hadoop-test/pom.xml   |    4 ++--
 .../cloudera/maven-packaging/hadoop-tools/pom.xml  |    4 ++--
 .../cloudera/maven-packaging/pom.xml               |    2 +-
 hadoop-mapreduce1-project/ivy/libraries.properties |    4 ++--
 13 files changed, 24 insertions(+), 24 deletions(-)

diff --git a/hadoop-mapreduce1-project/build.xml b/hadoop-mapreduce1-project/build.xml
index ae10ce5..7e1db05 100644
--- a/hadoop-mapreduce1-project/build.xml
+++ b/hadoop-mapreduce1-project/build.xml
@@ -54,7 +54,7 @@
  
   <property name="Name" value="Hadoop"/>
   <property name="name" value="hadoop"/>
-  <property name="version" value="2.2.0-mr1-cdh5.0.0-SNAPSHOT"/>
+  <property name="version" value="2.3.0-mr1-cdh5.0.0-SNAPSHOT"/>
   <property name="final.name" value="${name}-${version}"/>
   <property name="test.final.name" value="${name}-test-${version}"/>
   <property name="year" value="2009"/>
diff --git a/hadoop-mapreduce1-project/cloudera-pom.xml b/hadoop-mapreduce1-project/cloudera-pom.xml
index b5cc2bb..4767652 100644
--- a/hadoop-mapreduce1-project/cloudera-pom.xml
+++ b/hadoop-mapreduce1-project/cloudera-pom.xml
@@ -26,13 +26,13 @@ NOTE - This POM is used for Cloudera's build and packaging process.
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+    <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
     <relativePath>cloudera/maven-packaging/pom.xml</relativePath>
   </parent>
   
   <groupId>com.cloudera.cdh</groupId>
   <artifactId>hadoop-ant</artifactId>
-  <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+  <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
   <packaging>pom</packaging>
   
   <description>Maven wrapper for Ant Hadoop build</description>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-client/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-client/pom.xml
index e7bf651..2d4bf45 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-client/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-client/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+    <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-client</artifactId>
-  <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+  <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
   <packaging>jar</packaging>
 
   <description>Apache Hadoop Client</description>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml
index 5e8f665..7e2b643 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-core/pom.xml
@@ -23,13 +23,13 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+    <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-core</artifactId>
-  <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+  <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
   <packaging>pom</packaging>
   
   <name>hadoop-core</name>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-examples/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-examples/pom.xml
index abc0be8..4ff151e 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-examples/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-examples/pom.xml
@@ -24,13 +24,13 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+    <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-examples</artifactId>
-  <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+  <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
   <packaging>pom</packaging>
   
   <name>hadoop-examples</name>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-minicluster/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-minicluster/pom.xml
index 26fbaa7..c97dc8c 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-minicluster/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-minicluster/pom.xml
@@ -20,12 +20,12 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+    <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-minicluster</artifactId>
-  <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+  <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
   <packaging>jar</packaging>
 
   <description>Apache Hadoop Mini-Cluster</description>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-mrunit/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-mrunit/pom.xml
index d40f2e1..bb2de11 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-mrunit/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-mrunit/pom.xml
@@ -24,13 +24,13 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+    <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-mrunit</artifactId>
-  <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+  <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
   <packaging>pom</packaging>
   
   <name>hadoop-mrunit</name>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-streaming/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-streaming/pom.xml
index fe3b378..5e1eb04 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-streaming/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-streaming/pom.xml
@@ -24,13 +24,13 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+    <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-streaming</artifactId>
-  <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+  <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
   <packaging>pom</packaging>
   
   <name>hadoop-streaming</name>
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tarball/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tarball/pom.xml
index 274b7a8..b715ca0 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tarball/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tarball/pom.xml
@@ -24,14 +24,14 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+    <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-tarball</artifactId>
   <packaging>pom</packaging>
-  <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+  <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
   
   <name>hadoop-tarball</name>
   
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-test/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-test/pom.xml
index 11660fb..5534474 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-test/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-test/pom.xml
@@ -24,14 +24,14 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+    <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-test</artifactId>
   <packaging>pom</packaging>
-  <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+  <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
   
   <name>hadoop-test</name>
   
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tools/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tools/pom.xml
index 79a9b2e..c0c2083 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tools/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/hadoop-tools/pom.xml
@@ -24,14 +24,14 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>hadoop-root</artifactId>
-    <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+    <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
   
   <groupId>org.apache.hadoop</groupId>
   <artifactId>hadoop-tools</artifactId>
   <packaging>pom</packaging>
-  <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+  <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
   
   <name>hadoop-tools</name>
   
diff --git a/hadoop-mapreduce1-project/cloudera/maven-packaging/pom.xml b/hadoop-mapreduce1-project/cloudera/maven-packaging/pom.xml
index 365806b..d0a4bba 100644
--- a/hadoop-mapreduce1-project/cloudera/maven-packaging/pom.xml
+++ b/hadoop-mapreduce1-project/cloudera/maven-packaging/pom.xml
@@ -27,7 +27,7 @@
   
   <groupId>com.cloudera.cdh</groupId>
   <artifactId>hadoop-root</artifactId>
-  <version>2.2.0-mr1-cdh5.0.0-SNAPSHOT</version>
+  <version>2.3.0-mr1-cdh5.0.0-SNAPSHOT</version>
   <packaging>pom</packaging>
   
   <description>Hadoop root</description>
diff --git a/hadoop-mapreduce1-project/ivy/libraries.properties b/hadoop-mapreduce1-project/ivy/libraries.properties
index 42a9bf9..eb9f9fb 100644
--- a/hadoop-mapreduce1-project/ivy/libraries.properties
+++ b/hadoop-mapreduce1-project/ivy/libraries.properties
@@ -42,8 +42,8 @@ ftplet-api.version=1.0.0
 ftpserver-core.version=1.0.0
 ftpserver-deprecated.version=1.0.0-M2
 
-hadoop-common.version=2.0.0-cdh5.0.0-SNAPSHOT
-hadoop-hdfs.version=2.0.0-cdh5.0.0-SNAPSHOT
+hadoop-common.version=2.3.0-cdh5.0.0-SNAPSHOT
+hadoop-hdfs.version=2.3.0-cdh5.0.0-SNAPSHOT
 
 hsqldb.version=1.8.0.10
 
-- 
1.7.9.5

